cfgov/unprocessed/apps/youth-employment-success/js/input-view.js

```diff
@@ -0,0 +1,78 @@
+import { setInitFlag } from '../../../js/modules/util/atomic-helpers';
+
+const defaultProps = {
+  type: 'text'
+};
+const NODE_MISSING_ERROR = 'InputView expects to be initialized with an <input> node matching the supplied `type` prop';
+
+let UNDEFINED;
+
+/**
+ * Noop fallback for undefined event handlers
+ * @returns {undefined} undefined
+ */
+function noop() {
+  return UNDEFINED;
+}
+
+/**
+ * InputView
+ * Generic view for <input> elements
+ * @class
+ *
+ * @classdesc Initializes the organism.
+ *
+ * @param {HTMLNode} element The root DOM element for this view
+ * @param {Object} props Properties the view should be initialized with
+ * @returns {Object} The view's public methods
+ */
+function InputView( element, props = {} ) {
+  const finalProps = {
+    ...defaultProps,
```

Comment:
What's your browser support requirements for this app? Spread operator doesn't have IE11 support I believe, which we do deliver JS to. This likely will not be handled by webpack/babel. See the comment in [webpack.config.js ](https://github.com/cfpb/cfgov-refresh/pull/5176/files#diff-33ad45ad5b983390a0ca720f55f3d49eR45) `Use useBuiltIns: 'usage' and set `debug: true` to see what scripts require polyfilling.`. You can change the settings that way and when you build the script you'll see output in the Terminal of what should be polyfilled. We don't automatically include babel-polyfill because it adds polyfills for parts of third party dependencies that we never utilize, so adds unnecessary bloat. Sometimes it's easier to just use older style code structures for compatibility.